lib/core/httpRouter/routeHandler.js

```diff
@@ -56,7 +56,8 @@ class RouteHandler {
         protocol: 'http',
         url: message.url,
         headers: message.headers,
-        ips: message.ips
+        ips: message.ips,
+        verb: message.method
```

Comment:
IMHO we should lowercase the verb to make it case insensitive

Reply:
I asked myself that question and, truth is, I don't know what to do with it.
On one hand, lowercases won't make it case-insensitive, and it feels more standard to use the exact same value as the one provided by Node.js' HTTP interface. On the other hand, other values we provide are usually in lowercases...

Reply:
I would rather put the verb in lowercase to follow "our" convention but I won't block this PR for that detail